frontend/src/pages/ConnectedPages/Game/index.tsx

```diff
@@ -1,23 +1,77 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import Quizz from "./Quizz";
 import "./game.css";
 import Countdown from "./Countdown";
-import { IGameEnum } from "./types";
+import { IGameEnum, ITrack, ITraksRequest } from "./types";
 import Score from "./Score";
+import { RouteComponentProps, Redirect } from "react-router-dom";
 
-const Game: React.FC = () => {
+const Game: React.FC<RouteComponentProps> = ({ location }) => {
   const [step, setStep] = useState<IGameEnum>(IGameEnum.COUNTDOWN);
+  const [tracks, setTracks] = useState<ITrack[] | null>(null);
+  const [error, setError] = useState<boolean>(false);
   const [score, setScore] = useState<number>(0);
 
+  useEffect(() => {
+    // TODO: Request to get tracks and choices
+    if (location.search !== "") {
+      const idGame = location.search
+        .slice(1)
+        .split("gameId=")
+        .filter(el => el !== "")
+        .shift();
+      console.log(idGame);
+
+      // TODO: Request game tracks
+      // GET "/game/:id"
+      // const requestRoom = axios({
+      //   method: "GET",
+      //   url: "/game/:id"
+      // });
+      // data: ITrack[]
```

Comment:
We can delete this comment as the tracks will arrive through websockets

Reply:
Yes